include/crm/common/remote_internal.h

```diff
@@ -61,6 +61,20 @@ pcmk__is_guest_or_bundle_node(const pcmk_node_t *node)
            && (node->details->remote_rsc->container != NULL);
 }
 
+/*!
+ * \internal
+ * \brief Check whether a node is a Pacemaker Remote node of any kind
```

Comment:
still kind of unfortunate that a (Pacemaker) remote node is different from a Pacemaker Remote™ node but what can ya do lol, this is as clear as it gets

Reply:
Yeah I tried to make that point when I started, but it was deemed unimportant. It's entrenched now, but it could be revisited someday when all the worse things are taken care of ;)